tests/cell.py

```diff
@@ -15,6 +18,10 @@
 
 gdspy.library.use_current_library = False
 
+tmpdir = pathlib.Path(__file__).parent / 'tmp'
```

Comment:
Use [`tmpdir` from pytest](https://docs.pytest.org/en/6.2.x/tmpdir.html#the-tmpdir-fixture) instead. Take a look at the tests for [`GdsLibrary`](https://github.com/heitzmann/gdspy/blob/master/tests/gdslibrary.py) or [`GdsWriter`](https://github.com/heitzmann/gdspy/blob/master/tests/gdswriter.py) for examples.